app/views/shared/navigation/_breadcrumbs.html.erb

```diff
@@ -44,37 +44,41 @@
       <%= last_item[:label] %>
     </span>
   <% else %>
-    <% template_paths = [/^\/protocols\/\d+$/, /^\/label_templates\/\d+$/] %>
-    <% if template_paths.any? { |pattern| request.path.match?(pattern) }%>
-     <% last_breadcrumb_items = @breadcrumbs_items.pop(2) %>
-    <% else %>
-      <% last_item = @breadcrumbs_items.pop %>
-    <% end %>
+    <% last_item = @breadcrumbs_items.pop %>
```

Comment:
Hi
I think we can check if `item` has a `url` or not. if not we just render an simple `span` like the last item.
In that case we also add a CSS class to make it look disabled
Something like this maybe?
```
      <% if item[:url] %>
        <%= link_to(item[:label], item[:url],
                    class: "breadcrumbs-link",
                    title: item[:label]) %>
      <% else %>
        <%= content_tag(:span, item[:label],
                        class: "breadcrumbs-link",
                        title: item[:label]) %>
      <% end %>
 ```

Reply:
We also need to do the same in case the breadcrumbs are shortened `root > ... > link1 > link 2 > link 3` (with the dropdown).
[this line](https://github.com/scinote-eln/scinote-web/pull/5664/files#diff-00b0b88581e8475f6f80b9c63d977059f47b616197e601222b1b71143312c582R5)